File: Prafful-Bisht/DAY-9/Day-9-A.cpp
class Solution {
public:
    int reverse(int x) {
        int ans = 0;
        int lg;
        while(x!=0){
            lg = x%10;
            if((ans > INT_MAX/10)||(ans<INT_MIN/10)){
                return 0;
            }
            ans = ans*10 + lg;
            x =x/10;
        }
        return ans;
    }
};

File: Mukul Bhalla/Day 9/Day 9 A.py
class Solution:
    def reverse(self, x: int) -> int:
        x = list(str(x))
        x.reverse()
        if x[-1] == '-':
            x = [x[-1]] + x
            x.pop(-1)
        x = "".join(x)
        if -2**31 <= int(x) <= ((2**31)-1):
            return int(x)
        else:
            return 0
